regulations/static/regulations/js/source/views/comment/comment-confirm-view.js

```diff
@@ -29,8 +29,8 @@ var CommentConfirmView = Backbone.View.extend({
     this.interval = window.setInterval(
       function() {
         $.getJSON(url).then(function(resp) {
-          this.setPdfUrl(resp.pdfUrl);
           if (resp.trackingNumber) {
+            this.setPdfUrl(resp.pdfUrl);
```

Comment:
We want to make the PDF available separately from the tracking number so that users can download the final file even if regs.gov has an issue.